tests/manifests/default/kustomization.yaml

```diff
@@ -3,11 +3,12 @@ kind: Kustomization
 
 resources:
   - ../../../config/default
-  - ./controller_def_1.4.1.yaml
   - ./controller_def_1.4.2.yaml
   - ./controller_def_1.4.3.yaml
   - ./controller_def_1.4.4.yaml
+  - ./controller_def_1.5.0-rc1.yaml
```

Comment:
not related to this PR but I incorporated the latest numaflow controller and removed an old one